### ISSUE TITLE
Is this plugin support volatility 3.0？

### ISSUE THREAD

**Issue by MaxFuwin** [breppo/Volatility-BitLocker#1]

**Comment by bitdruid** [comment]
no, imports are wrong among other significant changes in volatility3. if you want to use it just create a python2 virtualenv and use old volatility with this plugin
